src/stylesheets/variables.scss

```diff
@@ -1,3 +1,5 @@
-$background-colour: #2E2D40;
-$background-gradient: radial-gradient(circle at 10% 20%, #373563 0%, transparent 50%),
-                      radial-gradient(circle at 90% 80%, #04022e 0%, $background-colour 40%);
\ No newline at end of file
+$background: linear-gradient(225deg, #2e2d40 0%, #3c3a61 100%);
+$background-circle1: linear-gradient(280deg, #3f3e5c 0%, #3a3c4a 100%);
```

Comment:
I can't see where `$background-circle1` and `$background-circle2` have been used?